pkg/cache/cache.go

```diff
@@ -868,6 +868,13 @@ func (c *Cache) MatchingClusterQueues(nsLabels map[string]string) sets.Set[strin
 	return cqs
 }
 
+func (c *Cache) ClusterQueueExists(cqName string) bool {
```

Comment:
It would be more accurate to obtain this from the `queue.Manager` instead, as that's what's involved in scheduling.

Reply:
That makes sense.

Reply:
Done.